test/MetaTx.js

```diff
@@ -0,0 +1,497 @@
+const lightwallet = require('eth-lightwallet')
```

Comment:
I'm not sure of the differences between the two, but it seems like a good idea to consolidate if possible. 

eth-lightwallet is essentially used currently to set up a vault and then derive a bunch of keys/accounts. I did a quick look though of eth-signer, and it does not look like this is possible. I might be missing something, though. 

@christianlundkvist is there any motivation here for using eth-lightwallet over eth-signer? Thanks :)